CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/StructuredGraphBuilderPlugin.java

```diff
@@ -56,6 +56,8 @@
     public static final String BACKBONE_DENSITY_PARAMETER_ID = PluginParameter.buildId(StructuredGraphBuilderPlugin.class, "backbone_density");
     public static final String RADIUS = PluginParameter.buildId(StructuredGraphBuilderPlugin.class, "radius");
     
+    private static final String NODE = "Node ";
```

Comment:
Maybe "~Node ", to be consistent with sphere graph?

Reply:
Done